scripts/form.js

```diff
@@ -77,6 +84,17 @@ const generateCard = (data) => {
   tagBox.append(tagButton);
 };
 
+// Reload Script
+
+function load_js() {
+  const head = document.querySelector('[data-js="head"]');
+  const script = document.createElement("script");
+  script.setAttribute("src", "");
+  script.setAttribute("src", "./scripts/answerbtn.js");
+  console.log("hello");
+  head.append(script);
+}
```

Comment:
This is only working with the first commited question. After that, the script stops working. Why?

Reply:
The reason is: the code in `answerbtn.js` will be run again. This leads to a try of redeclaration of the constant `answerBtn` which is not possible.

A possible solution is to include the `answerbtn.js` script on the form page.
Then wrap your code into a function:
```js
function answerButton() {
  const answerBtn = document.querySelectorAll('[data-js="answerBtn"]');

  const answer = document.querySelectorAll('[data-js="answer"]');
  // ...
```
Then, instead of loading the script once more into your page via `load_js()`, just call the function `answerButton()` which registers the event listeners.

The code could be optimised even more. Your function `generateCard()` could return the generated card which could then be passed into the function `answerButton(card)`. The eventListener and the toggle functionality can then be written specifically for that single card.

I hope this helps ;)